src/actions/actions.ts

```diff
@@ -2190,11 +2188,11 @@ class MoveToMatchingBracket extends BaseMovement {
       let matchedPosition: Position | undefined = undefined;
 
       for (const { char, pos } of Position.IterateDocument(position, toFind.nextMatchIsForward)) {
-        if (char === charToMatch) {
+        if (char === charToMatch && charToMatch !== toFind.match) {
           stackHeight++;
         }
 
-        if (char === this.pairings[charToMatch].match) {
+        if (char === toFind.match) {
```

Comment:
Good catch! That must have been vestigial.

I would also suggest adding ' and " back to our `pairings` list, perhaps adding a flag so that we don't match them with %.